util/src/main/java/com/ticketmaster/servos/util/DialogUtils.java

```diff
@@ -0,0 +1,102 @@
+package com.ticketmaster.servos.util;
+
+import android.app.AlertDialog;
+import android.app.AlertDialog.Builder;
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.DialogInterface.OnClickListener;
+import android.content.DialogInterface.OnShowListener;
+import android.text.Editable;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.EditText;
+
+public final class DialogUtils {
+
+    private DialogUtils() { }
+
+    /**
+     * Get a dialog that shows the provided title and an indeterminate spinner. You are responsible for showing and canceling it.
+     */
+    public static ProgressDialog getProgressDialog(Context context, int titleResId) {
```

Comment:
Recommendation:

I think it'd be nice to have versions of each of these constructors that can take either a String resource or a String object for title, message, hint text, etc. That way all cases are covered. Right now some take resources and some take Strings, but none of the dialogs can handle either one.